ui/src/main/js/view/widgets/dialog/style.less

```diff
@@ -1,13 +1,16 @@
 .cbwf-dialog {
   border-radius: @cbwf-border-radius;
   border: solid 1px @dark-border-color;
+  border: solid 1px var(--panel-border-color);
```

Comment:
I'm a big reluctant on using this variables bc I think they may need to be refactored / renamed in the future.

Reply:
I think this one should be okay?

Reply:
This one is actually an example of one that's problematic. This one is used for the side panel and the realtime widgets border. Then there's `--pane-border-color` that is used as border on `.pane-frame`.